spec/std/enumerable_spec.cr

```diff
@@ -1364,6 +1365,18 @@ describe "Enumerable" do
     it { [1, 2, 3].sum(4.5).should eq(10.5) }
     it { (1..3).sum { |x| x * 2 }.should eq(12) }
     it { (1..3).sum(1.5) { |x| x * 2 }.should eq(13.5) }
+    it { [1, 3_u64].sum(0_i32).should eq(4_u32) }
+    it { [1, 3].sum(0_u64).should eq(4_u64) }
+    it { [1, 10000000000_u64].sum(0_u64).should eq(10000000001) }
+    it "raises if union types are summed" do
```

Comment:
Specs that take a long time (e.g. run the compiler) should be marked as `slow`:
```suggestion
    it "raises if union types are summed", tags: %w[slow] do
```

Reply:
Done!